.changeset/cold-swans-share.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/octicons": patch
```

Comment:
```suggestion
"@primer/octicons": minor
```

Reply:
This should be a minor update because we are adding new icons. A patch would be used for something like when we update a current icon's vectors but it is still named the same.